### DISCUSSION TITLE
Is there a plan for the management console

### DISCUSSION THREAD

**Discussion by felixYyu** [zendesk/maxwell#2046] [General]
In the case of a large number of synchronized data tables, there is still a large threshold for configuring the whitelist and metadata troubleshooting. If you can provide a management console function and operate through the page, the promotion will be better，Is there a plan for follow-up?

**Comment by osheroff** [comment]
there are no active plans to put work into such a task.  PRs welcome, though.

**Comment by wanghangyu817** [comment]
![image](https://user-images.githubusercontent.com/18672877/178891789-3f09312b-e489-4b09-b866-5ce6133b7ac7.png)
![image](https://user-images.githubusercontent.com/18672877/178891831-fa06a0e8-ce7f-4ce8-bd33-4ee2543786dc.png)
![image](https://user-images.githubusercontent.com/18672877/178891917-9d53b520-738e-4a24-82f8-3b3216dc6df9.png)

- We have made a set of Maxwell's visual management system, including metadata management, one-click start and stop, monitoring and so on.  However, it still relies on a lot at present, so I tried to simplify it

**Comment by wanghangyu817** [comment]
@osheroff The question we will discuss is whether maxwell needs to have springBoot and other web frameworks built in, and whether this will make Maxwell too heavy.  Whether to make the console plug-in.I need your advice

**Comment by osheroff** [comment]
I don't speak the language in the screenshots so I'm really not sure what I'm looking at.  the features you list sound pretty cool, but I wouldn't want to take on a huge amount of dependencies.  what size are the dependencies?

Would it be possible to architect your frontend so that it "wraps" maxwell, instead of living inside it?  If that's the case then you could distribute your frontend separate from maxwell (I'd link to it) and they could be cleanly separated.

**Comment by wanghangyu817** [comment]
I tried to separate it from maxwell, which is now integrated because there's a lot of monitoring and logic in this system

**Comment by osheroff** [comment]
open a PR and I can look

**Comment by mmm8955405** [comment]
A year has passed, is this UI integrated?

**Comment by wanghangyu817** [comment]
We've been using it, but it hasn't been integrated into maxwell yet

**Comment by mmm8955405** [comment]
> We've been using it, but it hasn't been integrated into maxwell yet

Do you have any information on how to use this plugin?
